### DISCUSSION TITLE
How to manual focus when using Octo4a and Octolapse?

### DISCUSSION THREAD

**Discussion by stevempotter** [FormerLurker/Octolapse#819] [Q&A]
I am using a Huawei P20 Pro (android phone, Octo4a installed) to run Octoprint server and Octolapse, and take the timelapse movies. The focus keeps shifting and some frames are terribly blurry. That is with "Disable Autofocus" (in the Camera Settings on the phone) turned off. When it is switched on, ALL the frames are out of focus. I can't find anywhere to set the focus for manual focusing. If I go to Octolapse camera settings and try to check the Custom Image Preferences box, I get an error window:
__
**Unable To Enable Custom Preferences**
An unknown webcam server type 'unknown' is being used for the 'Webcam - Default OctoPi 0.16.0' camera profile. Currently only MJPEGStreamer is supported. Unable to apply custom image preferences.
__

How to manually set the focus of an android phone camera for use in Octolapse?

**Comment by AppleDeveloperBCN** [comment]
Hi, Have you find out the solution? I am in the same issue with an Honor X8....

**Comment by stevempotter** [comment]
Nope. Gave up on Octolapse.
